CSV_Trans_Col_To_Rec_.py

```diff
@@ -1,71 +1,71 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# vim:fenc=utf-8
+# vim:fenc=utf-8 
 #! /usr/bin/env python
 # csvファイル読み込みライブラリをImport
 import os
 import csv
 
 def trance(I_files,O_files,begin,column,repeat):
```

Comment:
According to [PEP-8](https://pep8-ja.readthedocs.io/ja/latest/), it's good to add spaces between arguments/parameters:  
```suggestion
def trance(I_files, O_files, begin, column, repeat):
```